dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,24 +1,16 @@
 module SurveyResultsHelper
   def show_survey?
-    # rubocop:disable Lint/UnreachableCode
-
-    # Disabled for all users on 2016-03-30 after being shown for thirty days.
-    return false
-
     # Reasons we would not show the survey.
     unless current_user
       return false
     end
-    unless current_user.teacher?
-      return false
-    end
     if current_user.under_13?
       return false
     end
     if language != "en"
       return false
     end
-    if SurveyResult.where(user_id: current_user.id).exists?
+    if SurveyResult.where({user_id: current_user.id, kind: "NetPromoterScore2017"}).exists?
```

Comment:
Ideally, we'd do something like
```
KINDS = [
  DIVERSITY_2016 = 'Diversity2016'.freeze,
  NET_PROMOTER_SCORE_2015 = 'NetPromoterScore2015'.freeze,
  NET_PROMOTER_SCORE_2017 = 'NetPromoterScore2017'.freeze
].freeze
validates :kind, inclusion: {in: KINDS}, allow_nil: false
```
in the `SurveyResult` model. Happy to do this in another PR, if you would like.